app/src/main/java/com/github/se/project/ui/lesson/SelectedTutorDetails.kt

```diff
@@ -0,0 +1,379 @@
+              // Tutor information section
+              TutorInfoSection(tutorProfile)
+
+              HorizontalDivider(color = MaterialTheme.colorScheme.outlineVariant)
+
+              // Lesson information section
+              TutorDescriptionSection(tutorProfile.description)
+
+              HorizontalDivider(color = MaterialTheme.colorScheme.outlineVariant)
+
+              // Tutor comments section
+              TutorCommentsSection(tutorProfile.rating.comments, listProfilesViewModel)
+            }
+      }
+}
+
+@Composable
+private fun TutorInfoSection(profile: Profile) {
+  Row(
+      modifier = Modifier.fillMaxWidth().testTag("tutorInfoRow"),
+      horizontalArrangement = Arrangement.spacedBy(16.dp),
+      verticalAlignment = Alignment.CenterVertically) {
+        // Profile picture placeholder
+        DisplayProfilePicture("tutorProfilePicture")
+
+        // Tutor details
+        Column(modifier = Modifier.weight(1f)) {
+          Text(
+              text = "${profile.firstName} ${profile.lastName}",
+              style = MaterialTheme.typography.titleMedium,
+              modifier = Modifier.testTag("tutorName"))
+          Text(
+              text = "${profile.section} - ${profile.academicLevel}",
+              style = MaterialTheme.typography.bodyMedium,
+              color = MaterialTheme.colorScheme.onSurfaceVariant,
+              modifier = Modifier.testTag("tutorAcademicInfo"))
+        }
+
+        // Price and rating
+        Column(horizontalAlignment = Alignment.End) {
+          DisplayRatingGrade(profile.rating.averageRating, "tutorRating")
+
+          Text(
+              text = "${profile.price}.-/h",
+              style = MaterialTheme.typography.labelMedium,
+              color = MaterialTheme.colorScheme.primary,
+              modifier = Modifier.testTag("tutorPrice"))
+        }
+      }
+}
+
+@Composable
+private fun TutorDescriptionSection(description: String) {
+  Column(
+      verticalArrangement = Arrangement.spacedBy(8.dp),
+      modifier = Modifier.fillMaxWidth().testTag("tutorDescriptionSection")) {
+        if (description.isEmpty()) {
+          Text(
+              text = "No description available.",
+              style = MaterialTheme.typography.bodyMedium,
+              modifier = Modifier.testTag("tutorDescriptionEmpty"))
+        } else {
+          Text(
+              text = description,
+              style = MaterialTheme.typography.bodyMedium,
+              modifier = Modifier.testTag("tutorDescription"))
+        }
+      }
+}
+
+@Composable
+private fun TutorCommentsSection(
+    comments: MutableList<Comment>,
+    listProfilesViewModel: ListProfilesViewModel
+) {
+  Column(
+      verticalArrangement = Arrangement.spacedBy(8.dp),
+      modifier = Modifier.fillMaxWidth().testTag("tutorCommentsSection")) {
+        Text(
+            text = "Best reviews on this tutor",
```

Comment:
Only showing the best reviews is maybe a little counterproductive, especially if the tutor has a lower rating. Maybe also showing some low ones would be useful.